cli/CHANGELOG.md

```diff
@@ -2,17 +2,49 @@
 
 This change log covers only the command line interface (CLI) of Open VSX.
 
+### v0.3.0 (Jan. 2022)
+
+#### Breaking Changes
+
+- The minimum version of Node.js required is now `14` because of the newer `vsce`
+
+#### Dependencies
+
+- Upgrade `vsce` from `1.97.0` to `2.6.3`
+- Upgrade `follow-redirects`
+- Upgrade `nth-check`
+
+---
+
+### v0.2.1 (Sep. 2021)
+
+#### New Features
+
+- Also accept `LICENSE` files (alternative spelling to `LICENSE`)
```

Comment:
The alternative spelling is `LICENCE`. Please include a PR link (https://github.com/eclipse/openvsx/pull/307)